package.json

```diff
@@ -27,7 +27,7 @@
   },
   "dependencies": {
     "@babel/core": "^7.10.4",
-    "@kiltprotocol/sdk-js": "^0.19.1-fe23e59.0",
+    "@kiltprotocol/sdk-js": "^0.19.1-be43462.0",
```

Comment:
I think this might be a problem since we are working with using the latest packages with our access token.